packages/webcert/src/feature/certificate/CertificateHeader/ShowHistory.tsx

```diff
@@ -46,7 +46,8 @@ const ShowHistory: React.FC<Props> = ({ historyEntries, certificateMetadata }) =
       case CertificateEventType.SIGNED:
         return 'Intyget är signerat'
       case CertificateEventType.AVAILABLE_FOR_PATIENT:
-        return 'Intyget är tillgängligt för patienten'
+        if (certificateMetadata.type === 'db' || certificateMetadata.type === 'doi') return
+        else return 'Intyget är tillgängligt för patienten'
```

Comment:
Should this perhaps be handled by backend instead, so that "AVAILABLE_FOR_PATIENT" never gets received for these types of certificates?